python/lsst/meas/astrom/astrometry.py

```diff
@@ -200,6 +201,7 @@ def solve(self, sourceCat, bbox, initWcs, filterName=None, calib=None, exposure=
 
         res = None
         wcs = initWcs
+        minDist = 0
```

Comment:
I introduce the minDist quantity (which should be maxDist by the way) to compute the new maximum distance between the source and the reference to be accepted as a match
This parameter should be propagated back matchOptimisticB::_doMatch